test/02-investigate.t

```diff
@@ -90,6 +90,9 @@ client-get-job() {
         10035)
             echo '{"job": { "test": "vim", "result": "failed", "parent_group": "foo", "group": "bar", "settings": {"OPENQA_INVESTIGATE_ORIGIN": "10035"} } }'
             ;;
+        10036)
```

Comment:
Are all these new fixtures necessary, when trigger_jobs is mocked?

Reply:
I didn't thought about it you are right! :100: